apps/explorer/src/index.css

```diff
@@ -14,7 +14,132 @@
     --steel-dark: theme('colors.steel.dark');
 }
 
-body {
-    @apply antialiased;
-    @apply font-sans text-offblack;
+@layer base {
+    body {
+        @apply antialiased;
+        @apply font-sans text-offblack;
+    }
+}
+
+@layer components {
+    .home-page-grid-container {
+        @apply grid grid-cols-1 gap-4;
+    }
+
+    @screen sm {
```

Comment:
Once we safely roll out the layout with the token card, I can clean up these duplicated grid layouts for all breakpoints